open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/MenuShortCut.java

```diff
@@ -0,0 +1,86 @@
+package io.opensphere.core.control.keybinding;
+
+import javafx.beans.property.SimpleStringProperty;
+
+/**
+ * Holds the information for entries within a JavaFX tableview used in the "Menu
+ * Shortcuts" tab of the "Help" -> "Key Map" Menu.
+ */
+public class MenuShortCut
+{
+
+    /**
+     * The category of the shortcut.
+     */
+    private final SimpleStringProperty myMenu;
+
+    /**
+     * The action the shortcut executes.
+     */
+    private final SimpleStringProperty myItem;
+
+    /**
+     * The physical key the shortcut is mapped onto.
+     */
+    private final SimpleStringProperty myShortcut;
+
+    /**
```

Comment:
All the methods in this class need descriptions

Reply:
updated slightly